intellihide.js

```diff
@@ -198,7 +199,7 @@ const Intellihide = new Lang.Class({
              * select a window in the secondary monitor.
              */
 
-            let monitorIndex = this._settings.get_int('preferred-monitor');
+            let monitorIndex = this._monitorIndex;
```

Comment:
This is an example of 'minimal chnage', where I tried to keep the implementation minimal. I guess we can simply remove lines 205-215 and use `this._monitorIndex` everywhere.